### ISSUE TITLE
Incorrectly constant propagation

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#60]
The following code was incorrectly optimized during the constant propagation phase
```
z := 20;
y := 13;
y := 7;
z := z * args->Size() + y;
z->PrintLine();
```

**Comment by objeck** [comment]
Issue was fixed in push commit eea07eddf6f8fbd8785f2c03b3fd40c72a0eb170. This commit includes dead store optimizations, which remove the offending code before the constant propagation phase.

**Comment by objeck** [comment]
Fixed in v6.6.2
